formats/fatality_notice/publisher/links.json

```diff
@@ -0,0 +1,17 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "fields_of_operation"
+  ],
+  "properties": {
+    "fields_of_operation" : {
+      "minItems": 1,
```

Comment:
@boffbowsh Reckon it should be named `field_of_operation` even though it's an array?